src/odemis/util/raster.py

```diff
@@ -142,3 +142,45 @@ def point_in_polygon(p, polygon):
         p1x, p1y = p2x, p2y
 
     return inside
+
+
+def get_possible_intersections(row_pairs: List[Tuple], col_pairs: List[Tuple], include_neighbors: bool = True) -> Set[Tuple]:
+    """
+    Uses Bresenham's line algorithm to determine the possible intersected fields/tiles.
+
+    :param row_pairs: (list of tuple) A list of (start_row, end_row) pairs defining line segments.
```

Comment:
tuples of what? Is that ints, or can they be float? What are the min/max values? It seems it can handle any large/small value, right?